### PR TITLE
Exercice 4 incomplet

### PR DESCRIPTION
**Pull request by jpcosal**

### CODE REVIEW

**Comment by Papelito83** [comment]
la frappe

File: Exam/src/exercice2/LogItemTest.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.junit.jupiter.api.Test;
+
+class LogItemTest {
+
+	@Test
+	void test() {
+		Date date = new Date();
+		LogItem log = new LogItem(date,"Niveau 1","Accepte");
+		System.out.print(log.format());
```
Comment (jpcosal) [inline Exam/src/exercice2/LogItemTest.java]: Why no assert??

File: Exam/src/exercice3/Jouet.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	private int nbRectangle;
+	private int nbTriangle;
+	
+	public Jouet(int nbCercle, int nbRectangle, int nbTriangle) {
+		this.nbCercle=nbCercle;
+		this.nbRectangle=nbRectangle;
+		this.nbTriangle=nbTriangle;
+	}
+	
+	public double AirCercle(int Rayon) {
```
Comment (jpcosal) [inline Exam/src/exercice3/Jouet.java]: Good, but you needed to dispatch this using abstract classes/interfaces and child classes

File: Exam/src/exercice4/Article.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package exercice4;
+
+public class Article {
```
Comment (jpcosal) [inline Exam/src/exercice4/Article.java]: The `Article` itself has no interest for the problem we're trying to solve.
Only the `Sale` concept is interesting.

**Pull request closed without merging**